.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_Store
```

Comment:
Could you please remove this line? This should be ignored in your `.gitignore_global` file.